itests/paych_api_test.go

```diff
@@ -197,7 +194,11 @@ func TestPaymentChannelsAPI(t *testing.T) {
 	require.Errorf(t, err, "Expected shortfall error of %d", excessAmt)
 
 	// wait for the settlement period to pass before collecting
-	waitForBlocks(ctx, t, bm, paymentReceiver, receiverAddr, policy.PaychSettleDelay)
+	head, err := paymentReceiver.ChainHead(ctx)
+	require.NoError(t, err)
+
+	// it doesn't matter which node we "wait" on
+	paymentReceiver.WaitTillChain(ctx, kit.HeightAtLeast(head.Height()+policy.PaychSettleDelay+5))
```

Comment:
what's up with `5`?

Reply:
Making sure that all nodes are at that height? Probably would be best to do as a helper, `kit.WaitTillChainAll..` or something like that

Reply:
@ZenGround0 Just a little extra buffer, not needed
@magik6k Yeah, good suggestion, will do